### PR TITLE
Support Flutter 3.10.0

### PR DESCRIPTION
**Pull request by MacDeveloper1**
- Add support for latest flutter 3.10.0.
- Use `flutter_lints` as `pedantic` deprerated

### CODE REVIEW

File: math_keyboard/pubspec.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 name: math_keyboard
-description: >-2
+description: >2-
   Math expression editing using an on-screen software keyboard or physical keyboard input in a
   typeset input field in Flutter.
-version: 0.1.9
+version: 0.10.0
```
Comment (edhom) [inline math_keyboard/pubspec.yaml:5]:
```suggestion
version: 0.2.0
```

**Comment by edhom** [comment]
@MacDeveloper1 thanks for the PR! We are quickly checking why the checks are failing:)

File: math_keyboard/pubspec.yaml [hunk 2]
```diff
@@ -17,30 +17,30 @@ dependencies:
 
   flutter_math_fork: ^0.6.3+1
```
Comment (edhom) [inline math_keyboard/pubspec.yaml:18]:
```suggestion
  flutter_math_fork: ^0.7.0
```

**Review by edhom — APPROVED** [review]
LGTM

**Comment by edhom** [comment]
@MacDeveloper1 Looks like your commits are not signed. To address the 3.10.0 issue, I changed the dependencies in the meantime in [this](https://github.com/simpleclub/math_keyboard/pull/53) PR. Would be cool if you could open another PR with signed commits to replace `pendantic` with `flutter_lints`!

**Comment by MacDeveloper1** [comment]
@edhom sorry for delay. Should this PR be closed?

**Pull request closed without merging**